crossbuild/cmake_toolchains/x86_64-unknown-freebsd.toolchain

```diff
@@ -0,0 +1,19 @@
+# Toolchain file for x86_64-unknown-freebsd
+set(CMAKE_SYSTEM_NAME FreeBSD)
+
+set(CMAKE_SYSROOT /opt/x86_64-unknown-freebsd/x86_64-unknown-freebsd/sys-root)
+set(CMAKE_INSTALL_PREFIX /)
+
+set(CMAKE_C_COMPILER /opt/x86_64-unknown-freebsd/bin/clang)
+set(CMAKE_CXX_COMPILER /opt/x86_64-unknown-freebsd/bin/clang++)
+set(CMAKE_LINKER /opt/super_binutils/bin/ld)
```

Comment:
Setting `CMAKE_LINKER` shouldn't be necessary; that `ld` is the first one on the path, as set [here](https://github.com/JuliaPackaging/BinaryBuilder.jl/blob/master/src/Runner.jl#L46-L47).

Reply:
Oh, there it is. I was trying to cross compile stuff in docker, and it didn't has this path setting. Since we assume this is used from BinaryBuilder.jl, this should be fine to be removed.